addons/trenches/stringtable.xml

```diff
@@ -32,8 +36,8 @@
             <Chinese>戰備鍬</Chinese>
         </Key>
         <Key ID="STR_ACE_Trenches_EnvelopeSmallName">
-            <English>Envelope - Small</English>
-            <German>Rund - Klein</German>
+            <English>Trench (Envelope - Small)</English>
+            <German>Schützengraben (Rund - Klein)</German>
             <Polish>Wnęka - mała</Polish>
             <Italian>Fascia - Piccola</Italian>
             <Spanish>Trinchera pequeña</Spanish>
```

Comment:
If you change a string, delete all other languages you don't speak or wrong information will be displayed for them!